org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorTemperatureAbs.java

```diff
@@ -0,0 +1,107 @@
+import com.zsmartsystems.zigbee.zcl.ZclAttribute;
+import com.zsmartsystems.zigbee.zcl.clusters.ZclColorControlCluster;
+import com.zsmartsystems.zigbee.zcl.clusters.colorcontrol.MoveToColorTemperatureCommand;
+import com.zsmartsystems.zigbee.zcl.protocol.ZclClusterType;
+
+import tech.units.indriya.unit.UnitDimension;
+
+/**
+ * Channel converter for color temperature, converting between the color control cluster and a Kelvin
+ * QuantityType<Temperature> channel.
+ *
+ * @author Andrew Fiddian-Green - Initial Contribution
+ */
+public class ZigBeeConverterColorTemperatureAbs extends ZigBeeConverterColorTemperature {
+
+    private Logger logger = LoggerFactory.getLogger(ZigBeeConverterColorTemperatureAbs.class);
+
+    private boolean updateStateDescriptionDone;
+
+    @Override
+    public void initialize(Channel channel, ZigBeeCoordinatorHandler coordinator, IeeeAddress address, int endpointId) {
+        super.initialize(channel, coordinator, address, endpointId);
+        updateStateDescriptionDone = false;
+    }
+
+    @Override
+    public void handleCommand(final Command command) {
+        if (command instanceof QuantityType<?> quantity && UnitDimension.TEMPERATURE == quantity.getDimension()) {
+            QuantityType<?> kelvin = quantity.toInvertibleUnit(Units.KELVIN);
+            if (kelvin != null) {
+                logger.debug("handleCommand() channel {} to {}", channelUID, kelvin);
+                MoveToColorTemperatureCommand zclCommand = new MoveToColorTemperatureCommand(
+                        1000000 / kelvin.intValue(), 10);
+                monitorCommandResponse(command, clusterColorControl.sendCommand(zclCommand));
+            }
+        } else {
+            super.handleCommand(command);
+        }
+    }
+
+    @Override
+    public Channel getChannel(ThingUID thingUID, ZigBeeEndpoint endpoint) {
+        return super.getChannel(thingUID, endpoint) == null ? null
+                : ChannelBuilder
+                        .create(createChannelUID(thingUID, endpoint,
+                                ZigBeeBindingConstants.CHANNEL_NAME_COLOR_TEMPERATURE_ABS),
+                                ZigBeeBindingConstants.ITEM_TYPE_NUMBER_TEMPERATURE)
+                        .withType(ZigBeeBindingConstants.CHANNEL_COLOR_TEMPERATURE_ABS)
+                        .withLabel(ZigBeeBindingConstants.CHANNEL_LABEL_COLOR_TEMPERATURE)
+                        .withProperties(createProperties(endpoint)).build();
+    }
+
+    @Override
+    public void attributeUpdated(ZclAttribute attribute, Object val) {
+        if (attribute.getClusterType() == ZclClusterType.COLOR_CONTROL
+                && attribute.getId() == ZclColorControlCluster.ATTR_COLORTEMPERATURE && val instanceof Integer mired) {
+            updateStateDescription();
+            updateChannelState(QuantityType.valueOf(1000000 / mired, Units.KELVIN));
+        } else {
+            super.attributeUpdated(attribute, val);
```

Comment:
>  you ALWAYS needs to call the base class - otherwise if someone uses the existing channel it will not work.

I am not so sure. 
1. The existing channel is an instance of the its own class, so for that class instance the handler always calls that method.
2. Whereas the new channel is an instance of a child class which wraps the parent class inside it. So for that (different) class instance the handler need only call the parent method to handle stuff that the child does not override.